### PR TITLE
Fix typos, code style and trim pictures in docs

### PR DESCRIPTION
**Pull request by michalfotyga**
Just fixed a typo and cut top part of two pictures that had too much white space there.

### CODE REVIEW

File: spring-batch-docs/asciidoc/common-patterns.adoc [hunk 1]
```diff
@@ -531,7 +531,7 @@ public Trade read() throws Exception {
             return t; // Record must end with footer
         }
     }
-    Assert.isNull(t, "No 'END' was found.");
+    Assert.notNull(t, "No 'END' was found.");
```
Comment (benas) [inline spring-batch-docs/asciidoc/common-patterns.adoc]: To be consistent, we need to update the code accordingly. But when I apply this change to `MultiLineTradeItemReader`, an `IllegalArgumentException` is thrown in `org.springframework.batch.sample.iosample.MultiLineFunctionalTests#testJob` (the test still passes though..). This is out of scope of this PR, so please revert this change for now. The rest LGTM.

**Comment by michalfotyga** [comment]
Nice catch @benas 👏 I have reverted that part.

**Comment by benas** [comment]
LGTM now. Thank you!
